topostats/validation.py

```diff
@@ -579,7 +572,7 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                     "Invalid value in config 'grain_image.image_type', valid values " "are 'binary' or 'non-binary'"
                 ),
             ),
-            "core_set": True,
+            "core_set": False,
```

Comment:
Might need to update `docs/configuration.md` to reflect this default value, sorry.